crates/hir_ty/src/tests/traits.rs

```diff
@@ -1475,7 +1475,6 @@ fn test(
 }
 
 #[test]
-#[ignore]
 fn error_bound_chalk() {
```

Comment:
This one looks like it behaves as I'd expect it to?

Reply:
Hm, seems so :shrug: I think we switched to leaving out error bounds completely at some point.